cv-data-controller/src/main/java/com/trihydro/cvdatacontroller/controller/ActiveTimController.java

```diff
@@ -111,6 +111,12 @@ public ResponseEntity<List<TimUpdateModel>> GetExpiringActiveTims() {
 				if (!rs.wasNull() && frameTypeValue >= 0 && frameTypeValue < TravelerInfoType.values().length) {

```

Comment:
Why is the frame type value unusable in the first place? Are we not saving it in the database when the TIM is initially created?

Reply:
Opened a work item for this here: https://dev.azure.com/trihydro/CV/_workitems/edit/14848